distance_logging.py

```diff
@@ -0,0 +1,59 @@
+
+from gamestate import game
+
+import memory
+import vars
+import threading
+import logging
+import json
+import time
+
+logger = logging.getLogger(__name__)
+
+
+game_vars = vars.vars_handle()
+
+def perform_distance_logging():
+    distances = {}
+    
+    def _update(distance_travelled, location, zone):
+        if location in distances:
+            cur_loc = distances[location]
+        else:
+            cur_loc = {}
+        if zone not in cur_loc:
+            cur_loc[zone] = 0
+        cur_loc[zone] += distance_travelled
+        distances[location] = cur_loc
+        logger.manip(f"Updated {location} - {zone} to {cur_loc[zone]}")
+    
+    def _monitor():
+        distance_travelled = 0
+        location = -1
+        zone = -1
+        previously_added = -1
+        last_updated_map_time = time.time()
+        while not (game.state == "Sin" and game.step == 4):
+            new_travelled = memory.main.total_distance_travelled()
+            if (not new_travelled) or (memory.main.get_map() != location) or (memory.main.get_zone() != zone):
+                if not distance_travelled or (distance_travelled == previously_added):
+                    pass
+                else:
+                    logging.manip(f"Adding {distance_travelled}")
```

Comment:
This uses logging.manip instead of logger.manip (the local annotated logger). I assume this is not intentional?